### PR TITLE
Scripts/SQL/Core: Implement Angon + item depletion for Repair/Tomahawk.

### PR DESCRIPTION
**Pull request by Hozu**
Scripted Angon, inflicts 20% defense down, requires Angon item to be equipped.
Changed the merit power of Angon to 15 for the duration (15 + merit power for 30-90 sec depending on investment).
Gave the proper model for the Angon item, as well as the Throwing Tomahawk.
Made the Angon/Throwing Tomahawk/Automaton Oils deplete upon using their respective ability (Tomahawk ability is currently unscripted so I merely put in the framework for it - whoever scripts it has to make sure to have the ammo ID check in OnAbilityCheck).

Edit: Thanks to L0ki for giving me a command to change equip models on the fly.

### CODE REVIEW

File: src/map/ai/ai_char_normal.cpp [hunk 1]
```diff
@@ -2248,19 +2248,18 @@ void CAICharNormal::ActionJobAbilityFinish()
 
         // TODO: все перенести в скрипты, т.к. система позволяет получать указатель на питомца
 
-        if (m_PJobAbility->getID() == ABILITY_CALL_BEAST || m_PJobAbility->getID() == ABILITY_REWARD)
+        if (m_PJobAbility->getID() == ABILITY_CALL_BEAST || m_PJobAbility->getID() == ABILITY_REWARD || m_PJobAbility->getID() == ABILITY_ANGON || m_PJobAbility->getID() == ABILITY_TOMAHAWK || m_PJobAbility->getID() == ABILITY_REPAIR)
```
Comment (teschnei) [inline src/map/ai/ai_char_normal.cpp]: multi line it, that is waay too long for a single line

Comment (Hozu) [inline src/map/ai/ai_char_normal.cpp]: Did I do that correctly?

File: scripts/globals/abilities/angon.lua [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    end
+end;
+
+-----------------------------------
+-- onUseAbility
+-----------------------------------
+
+function onUseAbility(player,target,ability)
+    local typeEffect = EFFECT_DEFENSE_DOWN;
+    duration = 15 + player:getMerit(MERIT_ANGON); -- This will return 30 sec at one investment because merit power is 15.
```
Comment (TeoTwawki) [inline scripts/globals/abilities/angon.lua]: Forgot to local your duration declaration :P

Comment (Hozu) [inline scripts/globals/abilities/angon.lua]: Good catch, I didn't notice because it gave me the correct effect duration when I used a print.

**Pull request merged**